### PR TITLE
[COLIB-NGC] add govHydro4

### PR DESCRIPTION
**Pull request by martinscripts**

### CODE REVIEW

File: pages/models/regulations/governor/GovHydro4/index.md [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+---
+
+## Context
+
+The following information has been gathered from [1], [2] and [3].
+
+In a hydroelectric power plant, the governing system’s main function is
+to control the speed or load of the turbine by the means of gate
+opening. The prime movers convert the kinetic energy of the water into
+mechanical energy, which is then converted into electrcial energy by a
```
Comment (Lampros-Cresym) [inline pages/models/regulations/governor/GovHydro4/index.md:19]: typo "electrical"

File: pages/models/regulations/governor/GovHydro4/index.md [hunk 2]
```diff
@@ -0,0 +1,340 @@
...
+Per-unit parameters are on base of $$P_\mathrm{base}$$, which is normally
+the capability of the turbine in MW. Per-unit angular velocity
+parameters are based on nominal frequency.
+
+<div id="tbl-parameters">
+
+Table 2: Parameters
+</div>
+
+| name                                   | type  | unit | IEC name | description                                                             | typical value         |
```
[the remark this answers is not in the document]
Comment (martinscripts) [reply pages/models/regulations/governor/GovHydro4/index.md:179]: I added refs to the table "Turbine type dependent parameters"

File: pages/models/regulations/governor/GovHydro4/index.md [hunk 3]
```diff
@@ -0,0 +1,340 @@
...
+<a href="#fig-control_diagram" class="quarto-xref">Figure 2</a>,
+involves rate feedback or transient gain reduction, which slows down or
+limits gate movement until water flow and power output have had time to
+adjust. As the system stabilizes, the governor returns to normal
+operation with permanent droop $$R_\mathrm{perm}$$, responding more
+sensitively to speed deviations to maintain steady-state operation. This
+compensation mechanism ensures stable control performance, crucial for
+maintaining consistent power output in hydroelectric power plants [1].
+
+Note: There is a difference between the intentional deadband and the
```
[the remark this answers is not in the document]
Comment (martinscripts) [reply pages/models/regulations/governor/GovHydro4/index.md:150]: moved this content to a dedicated section instead

**Comment by Lampros-Cresym** [comment]
I am not aware of the term “clattering”. It might be that we are saying the same thing in different words 😊

Nevertheless, to give a bit more of background, here is one reference talking about the backlash. https://ceeesa.es.anl.gov/projects/psh/ANL_DIS-13_05_Review_of_Existing_Hydro_and_PSH_Models.pdf
The model is a bit different compared to ours, but the governor part is largely the same.
***@***.******@***.******@***.***

There is a lot of information on backlash and how to model it here: https://www.diva-portal.org/smash/get/diva2:1050397/FULLTEXT01.pdf

**Comment by martinscripts** [comment]
Now that I read it maybe it is a mistranslation from German. I will think about it!

**Comment by martinscripts** [comment]
yes, actually. Backlash is exactly what I wanted to say. thanks

**Review by Lampros-Cresym — APPROVED** [review]

**Pull request merged**